fleet_adapter_template/fleet_adapter_template/fleet_adapter.py

```diff
@@ -215,9 +218,13 @@ def navigate(self, destination, execution):
             f'on map [{destination.map}]'
         )
 
+        self.node.get_logger().info(f'Converting RMF to Robot coordinates...')
+        rmf_pos = destination.position
+        robot_pos = self.rmf_to_robot_transforms[destination.map].apply(rmf_pos)
```

Comment:
Same as below, in the EasyFullControl code this transform [is applied](https://github.com/open-rmf/rmf_ros2/blob/d3f6bcf25267f9adaf9a41f074a919b6292300c8/rmf_fleet_adapter/src/rmf_fleet_adapter/agv/EasyFullControl.cpp#L1330) to the position before it is sent to this callback